### ISSUE TITLE
Q: do we need to generate a dummy DSIG table for noto fonts?

### ISSUE THREAD

**Issue by marekjez86** [googlefonts/fontmake#678]
All of the Noto fonts get FAIL when I run fontbakery which claims:
=======================
FAIL: This font lacks a digital signature (DSIG table). Some applications may require one (even if only a dummy placeholder) in order to work properly. You can add a DSIG table by running the `gftools fix-dsig` script. [code: lacks-signature]
=======================

do we need to generate a dummy DSIG table for noto fonts?
I don't want to run gftools fix-dsig on 2k+ fonts

**Comment by felipesanches** [comment]
yes, it would be good to have such dummy DSIG added to fonts by default so that we would not need to perform post-processing manually.

Such post-processing is possible nowadays by running the mentioned `gftools fix-dsig` script.

**Comment by chrissimpkins** [comment]
See https://github.com/googlefonts/fontmake/issues/431
